src/components/tasks/card/index.tsx

```diff
@@ -203,6 +270,26 @@ const Card: FC<Props> = ({
           </span>
         </span>
       </div>
+      <div className={`${classNames.taskTagLevelWrapper} ${shouldEdit && classNames.editMode}`}>
+            <div className={classNames.taskTaglevelContainer}>
+            {
+              taskTagLevel?.map((item) => (
+                <span key={item.itemid} className={classNames.taskTagLevel}>{item.tagname} <small><b>LVL:{item.levelname}</b></small>{
+                  shouldEdit && isUserAuthorized &&
+                  (
+                    <span>
+                    <button className={classNames.removeTaskTagLevelBtn} onClick={
+                      () => updateTaskTagLevel(item,'delete')
+                      }>&#10060;</button>
+                </span>
```

Comment:
I believe this is according to design

Reply:
you mean design for the UI?

Reply:
Yeah like LVL: and all

Reply:
design by me, open to ideas. @harshith-venkatesh